docs/source/en/_toctree.yml

```diff
@@ -42,7 +42,8 @@
       title: Use tokenizers from 🤗 Tokenizers
     - local: multilingual
       title: Inference for multilingual models
-    - sections:
+    - isExpanded: false
```

Comment:
this should not be changed in this PR

Reply:
I agree, but make style changes it and is required for code quality, will find a fix.

Reply:
That's weird, I don't locally have `make style` do any changes to the ToC once the model is in it.